src/main/java/felix/utils/Ui.java

```diff
@@ -0,0 +1,85 @@
+package felix.utils;
+
+import java.util.Scanner;
+
+public class Ui {
+    private static final String BOT_NAME = "Felix";
+    private static final String LOGO = "___________    .__  .__        \n"
+            + "\\_   _____/___ |  | |__|__  ___\n"
+            + " |    __)/ __ \\|  | |  \\  \\/  / \n"
+            + " |     \\\\  ___/|  |_|  |>    <  \n"
+            + " \\___  / \\___  >____/__/__/\\_ \\ \n"
+            + "     \\/      \\/              \\/ \n";
+    private static final String LINE_SEPARATOR = "__________________________"
```

Comment:
Suggestion to make `LINE_SEPARATOR` size explicit: `LINE_SEPARATOR = "_".repeat(60);`

Reply:
Thank you for the feedback! I will fix this.